mediapipe/graphs/instantmotiontracking/calculators/gl_animation_overlay_calculator.cc

```diff
@@ -402,6 +309,67 @@ bool GlAnimationOverlayCalculator::LoadAnimationAndroid(
       LOG(ERROR) << "Failed to read indices for frame " << frame_count_;
       return false;
     }
+
+    // Set triangle_mesh normals for shader usage
+    int len = lengths[2];
+    triangle_mesh.normals.reset(new float[len]);
```

Comment:
We might start by calculating one normal for each triangle, but eventually we need to give to the shader one normal for each vertex (unless we do something fancier like use a texture for our normals).

So we probably want this length to be lengths[0], while the temporary array "triangle_normals" below should stay lengths[2].